src/main/resources/InstructorSampleData.json

```diff
@@ -2763,23 +2763,111 @@
     }
   },
   "feedbackResponseComments": {
-    "comment1FromT1C1ToR1Q1S1C1": {
+    "comment1FromT1C1ToR1Q2S1C1": {
       "courseId": "demo.course",
       "feedbackSessionName": "Session with different question types",
       "feedbackQuestionId": "2",
       "giverEmail": "teammates.demo.instructor@demo.course",
-      "giverSection": "None",
-      "receiverSection": "None",
+      "giverSection": "Tutorial Group 1",
+      "receiverSection": "Tutorial Group 1",
       "feedbackResponseId": "2%alice.b.tmms@gmail.tmt%alice.b.tmms@gmail.tmt",
       "showCommentTo": [
-        "GIVER"
+        "GIVER",
+        "RECEIVER",
+        "RECEIVER_TEAM_MEMBERS",
+        "INSTRUCTORS"
+      ],
+      "showGiverNameTo": [
+        "GIVER",
+        "RECEIVER",
+        "RECEIVER_TEAM_MEMBERS",
+        "INSTRUCTORS"
       ],
-      "showGiverNameTo": [],
       "isVisibilityFollowingFeedbackQuestion": false,
-      "createdAt": "2016-03-01 11:59 PM UTC",
+      "createdAt": "2017-06-03 9:17 PM +0530",
```

Comment:
Missed this - maybe we should leave all times at UTC (the course is already at UTC).